### DISCUSSION TITLE
student commenting behavior

### DISCUSSION THREAD

**Discussion by amitvtcbb** [community/community#62136] [GitHub Education] [answered]
Hello Everyone,

I am Amit Patel. I am working as Assistant Professor in BCA Department to Vidyabharti Trust College of BCA affiliated to Veer Narmad South Gujarat University.

I am pursuing my Ph.D and my topic is related to "Modeling Student Behavior" for programming Assignment. I this topic , we want to modelling student behavior regarding programming assignment using GitHub.

For this purpose, I want student communicate on GitHub through their assignment. so I create my subject programming assignment on "GitHub Classroom" and share to student. Student have to submit their assignment and also make comment on other assignment.

I need your suggestion , regarding how to motivate students to put comments on each other to solve their assignment problem. and also want your help regarding collect comment log from your institution from students so I complete my research work.

Please contact me on _[removed by moderator]_ if any suggestion or query.

**Comment by tuves** [comment]
Welcome to the GitHub Community, @amitvtcbb, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. 

I've also removed your private information from [this post](link to discussion) as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Reply by amitvtcbb** [reply, marked as answer]
Thank you @tuves
